package.json

```diff
@@ -56,5 +56,8 @@
     },
     "resolutions": {
         "react-virtualized": "9.22.3"
+    },
+    "dependencies": {
+        "applicationinsights": "2.5.1"
```

Comment:
I recommend moving this to the `ide-extension` package instead of the root since it is only used in the extension module.